node/src/neighborhood/mod.rs

```diff
@@ -5513,7 +5518,7 @@ mod tests {
             database: &mut NeighborhoodDatabase,
             agrs: Vec<AccessibleGossipRecord>,
             gossip_source: SocketAddr,
-            _peer_addrs: &[IpAddr],
+            _connection_progress_peers: &[IpAddr],
```

Comment:
This field shouldn't be ignored by the mock, and whatever tests assert on these parameters should also assert on this one.